model/Core/Classes/Dictionary.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DictionaryEntry
+
+## Summary
+
+A key with an associated value.
+
+## Description
+
+The class used for implementing a generic dictionary in SPDX.  A dictionary consists of a collection of key-value pairs. Each key-value pair maps the key to its associated value.  The DictionaryEntry is the type used for the generic key-value pair.  Both the key and the value are strings.
```

Comment:
I suggest changing "implementing a generic dictionary" to "implementing a generic string mapping"

Reply:
We should also mention (copying from wikipedia):

> associative arrays, and a data structure more commonly known today as a dictionary or hash table.

Reply:
I re-wrote the description to address the above comments as well as @maxhbr comment below.

I'm not very happy with the rewrite - but I can't come up with clearer wording that incorporates the feedback.

Please feel free to propose and specific language changes.